### PR TITLE
fixes #8467 - trends range selector is now visible

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by ohadlevy** [comment]
screenshot from 2014-11-20 21 40 22

VS
screenshot from 2014-11-20 21 41 06

File: app/helpers/trends_helper.rb [hunk 1]
```diff
@@ -14,7 +14,9 @@ def trendable_types(new_record)
 
   def trend_days_filter
     form_tag @trend, :id => 'days_filter', :method => :get, :class=>"form form-inline" do
-      content_tag(:span, (_("Trend of the last %s days.") % select(nil, 'range', 1..Setting.max_trend, {:selected => range}, {:class=>"col-md-1", :onchange =>"$('#days_filter').submit();$(this).disabled();"})).html_safe)
+      content_tag(:span, (_("Trend of the last %s days.") %
```
Comment (shlomizadok) [inline app/helpers/trends_helper.rb:17]:
``` ruby
content_tag(:span, (_("Trend of the last %{days_select} days.") %  { :days_select => .....
```

Comment (ohadlevy) [inline app/helpers/trends_helper.rb:17]: days_select is not useful as max_trend can be bigger than 31 days.

**Comment by dLobatog** [comment]
Merged as 50594a5d105f53b0321e86839b2d72b26bf4a24e, thanks @ohadlevy ! Works well with > 30 days ranges and it looks nicer.

**Pull request closed without merging**